pycmr/queries.py

```diff
@@ -198,10 +289,21 @@ def query(self):
         options_as_string = "&".join(formatted_options)
 
         url = "{}?{}&{}".format(self.base_url, params_as_string, options_as_string)
-
+        print(url)
         response = get(url)
         return response.json()
```

Comment:
Although I suspect this will need to be broken up when we get into error handling (#20)